### ISSUE TITLE
Farmer summary can't find remote harvester, but they are connected when checking on INFO logs

### ISSUE THREAD

**Issue by ThalesC** [Chia-Network/chia-blockchain#3875]
I have a farmer+node+wallet, with ip 10.1.2.180, setup on one computer and a harvester, with ip 10.1.2.181, on another computer, both running linux.

Following the guide at [https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines](url) I was able to check on the logs of both machines that they can talk to each other.

Log on harvester:
```
2021-05-05T21:54:36.436 harvester chia.harvester.harvester: INFO     0 plots were eligible for farming b896c42767... Found 0 proofs. Time: 0.00064 s. Total 2 plots
2021-05-05T21:54:36.437 harvester harvester_server        : INFO     -> farming_info to peer 10.1.2.180 eb9a0a7c734858cebd587c02969acfd8b343095d54b6f88d0e8b1bf7d0b888de
```
Log on farmer:
```
2021-05-05T21:56:24.513 farmer farmer_server              : INFO     <- new_signage_point from peer 6619c645bc934cb8aa5bdc7bd326481d929f69e1e53de7170894a222ccc62917 127.0.0.1
2021-05-05T21:56:24.513 farmer farmer_server              : INFO     -> new_signage_point_harvester to peer 10.1.2.181 cabbc747656cc54a8f20dbec5da3dab10e2ccb5ac888bca1a8f696c8acb140c4
2021-05-05T21:56:24.517 farmer farmer_server              : INFO     <- farming_info from peer cabbc747656cc54a8f20dbec5da3dab10e2ccb5ac888bca1a8f696c8acb140c4 10.1.2.181
```
I wasn't sure if it's working properly but according to the guide I should see what I am currently seeing on my farmer logs.

Running `chia farm summary` on the farmer machine gives me the following output:
```
Connection error. Check if harvester is running at 8560
Farming status: Farming
Total chia farmed: 0.0
User transaction fees: 0.0
Block rewards: 0.0
Last height farmed: 0
Plot count: Unknown
Total size of plots: Unknown
Estimated network space: 2391.269 PiB
Expected time to win: Unknown
Note: log into your key using 'chia wallet show' to see rewards for each key
```

I am very confused, is this setup working just looking at the logs?
Thanks in advance!

**Comment by kswtch** [comment]
Everything is working in your setup. 

The patch to include connected harvesters on the farmers output of `chia farm summary` is not yet live. Until then you have to check your harvesters logs if it is talking to your full node.

**Comment by ThalesC** [comment]
Thank you @kswtch!
I feel less confused now, but having `chia farm summary` tell if it's working would be nice, I guess we just wait then.

**Comment by twitchplaystrading** [comment]
I have the exact setup and confirmed it's correctly set up on both machines but it's been farming 2 whole days without progress. Tried restarting the client, restarting the machines, resetting the router.

**Comment by ThalesC** [comment]
Mine is running for some days now with 26 k32 plots, somewhere around 2.6TB, and my harvester recognizes all of them and the new ones that pop, but received no rewards.
I don't know if this is expected for this quantity/size of plots.

**Comment by mariano54** [comment]
Fixed by 1.2.0
